submissions/SergSenras/tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,54 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: _put repo URL here_
+  Web app: _put project's github pages URL here_
+*/
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Dweller {
+  constructor (species,name,gender,saying,friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  getInfo() {
+    var str = '';
+    for (var key in this) {
```

Comment:
The order of props in an object is not guaranteed. Please, list props explicitly.
Hint: `[varA, varB].join()`
You may also want to enjoy `Array.map` to convert string keys into strings that both mention the key and associated value.
`["keA","keyB"].map().join()`

Reply:
```suggestion
  getInfo() {
    return ['species','name','gender','hands','legs','saying','friends']
      .map( x => '<b>' + x + '</b>: ' + this[x] )
      .join('; ');
  }
```
tried to do something similar to your description but still not sure